### DISCUSSION TITLE
为什么下载完没有 getClient 方法

### DISCUSSION THREAD

**Discussion by bear-qin** [w7corp/easywechat#2523] [General]
composer require w7corp/easywechat  下载之后的代码 和github上的不一样，没有application 类，没有getClient方法

**Comment by overtrue** [comment]
你试试放个截图，安装记录啥的？

**Reply by bear-qin** [reply]
![1656497373(1)](https://user-images.githubusercontent.com/101318059/176411532-e2ef1dc2-0bea-4b3b-a9ae-5727c6e6a7a4.png)

![1656497426(1)](https://user-images.githubusercontent.com/101318059/176411796-bb489e7c-d036-4a1a-9114-8a8f1c14a2d5.png)

**Reply by bear-qin** [reply]
![1656497373(1)](https://user-images.githubusercontent.com/101318059/176411532-e2ef1dc2-0bea-4b3b-a9ae-5727c6e6a7a4.png)

![1656497426(1)](https://user-images.githubusercontent.com/101318059/176411796-bb489e7c-d036-4a1a-9114-8a8f1c14a2d5.png)

**Reply by bear-qin** [reply]
![1656497628(1)](https://user-images.githubusercontent.com/101318059/176412392-79066bac-7b13-45b5-83b4-03ce1b201cec.jpg)

**Reply by bear-qin** [reply]
我的镜像原  是 腾讯云的镜像，为什么下载来的代码 和github不一样

**Reply by bangbangda** [reply]
用官方源安装，你这个是5.x的版本，不是6.x的版本。

**Comment by overtrue** [comment]
```bash
composer require w7corp/easywechat:^6.0
```
